website/components/LandingPage/CodeExample.tsx

```diff
@@ -1,62 +1,62 @@
 import { useSnapshot } from "valtio";
 import Highlight, { defaultProps } from "prism-react-renderer";
 import { GettingStarted } from "./GettingStarted";
-import { animationState, incDuration, decDuration } from "./animationState";
+import { state, incDuration, decDuration } from "./state";
 
-const exampleCode = (duration: number, count: number) => `
-  const animationState = proxy({
-    durationInSec: ${duration},
+const exampleCode = (dur: number, count: number) => `
+  const state = proxy({
+    dur: ${dur},
     count: ${count}
   });
-  const incDuration = () => {
-    ++animationState.durationInSec;
-  };
-  const decDuration = () => {
-    --animationState.durationInSec;
+  const incDur = () => {++state.dur};
+  const decDur = () => {--state.dur};
+  const incCount = () => {
+    ++state.count;
+    setTimeout(incCount, 100 * state.dur);
   };
 
-  setInterval(() => {
-    ++animationState.count;
-  }, 100);
+  incCount();
+
+  const snap = useSnapshot(state)
   
-  <div>
-    <h3>
-      {snapshot.durationInSec}
-    </h3>
-    <button
-      disabled={snapshot.durationInSec <= 1}
-      onClick={decDuration}>
-      -
-    </button>
-    <button
-      disabled={snapshot.durationInSec >= 10}
-      onClick={incDuration}>
-      +
-    </button>
-  </div>
+  return (
+    <div>
+      <h3>{snap.dur}</h3>
+      <button 
+        disabled={snap.dur <= 1}
+        onClick={decDur}>
+        -
+      </button>
+      <button
+        disabled={snap.dur >= 10}
+        onClick={incDur}>
+        +
+      </button>
+    </div>
+  );
 `;
 
 export const CodeExample = () => {
-  const snapshot = useSnapshot(animationState);
+  const snapshot = useSnapshot(state);
```

Comment:
Change this too for consistency?
```suggestion
  const snap = useSnapshot(state);
```

Reply:
roger that